### PR TITLE
migrate e2e vamc left nav test to cypress

### PR DESCRIPTION
**Pull request by brangi**
## Description
migrate e2e vamc left nav test to cypress and disable cypress tests against content on circle

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Comment by brangi** [comment]
@U-DON , can you help reviewing this? I don't know what's the best approach to skip the cypress tests on circleCI but I hate to see them fail.

File: script/run-cypress-tests-circle.sh [hunk 1]
```diff
@@ -11,5 +11,5 @@ else
   reporterArgs="--reporter cypress-multi-reporters --reporter-options \"configFile=config/cypress-reporters.json\""
 
   # Start the web server & run Cypress tests.
-  yarn start-server-and-test "node src/platform/testing/e2e/test-server.js --buildtype=vagovprod --port=3001 > /dev/null 2>&1" :3001 "yarn cy:run $reporterArgs"
+  yarn start-server-and-test "node src/platform/testing/e2e/test-server.js --buildtype=vagovprod --port=3001 > /dev/null 2>&1" :3001 "yarn cy:run $reporterArgs --env ci_name=circle"
```
Comment (U-DON) [inline script/run-cypress-tests-circle.sh:14]: I will be exposing the already available `CIRCLECI` environment variable, so you can revert the addition of this flag. #14113

Comment (brangi) [reply script/run-cypress-tests-circle.sh:14]: 👏 nice, will do.

**Comment by mpelzsherman** [comment]
This looks great. Can we add a check that the side nav collapses into a menu on mobile? That also could be done in a separate PR I guess.

Seems to me we could use this same approach for testing other aspects of the VAMS pages (e.g. breadcrumbs).

**Review by mpelzsherman — APPROVED** [review]
This looks great. Can we add a check that the side nav collapses into a menu on mobile? That also could be done in a separate PR I guess.

Seems to me we could use this same approach for testing other aspects of the VAMS pages (e.g. breadcrumbs).

**Review by U-DON — APPROVED** [review]

**Pull request merged**